website/blog/2018-01-10-1.10.0.md

```diff
@@ -0,0 +1,373 @@
+
+### Open source as contributors
+
+For the first 6 months of the project, I was working full time on it and ended up writing only half of the commits. This is pretty mind-blowing in itself! Not only was the throughput two times better on the project itself, but people ended up working on things that I wouldn't have spent time on like integrations with all the editors on earth, a fuzzer to find bugs, TypeScript support, infrastructure to support parsing multiple languages in the same file, and more.
+
+Even though we don't use all those things at Facebook, many of them ended up being useful. CSS-in-JS support made it easy to get GraphQL fragments in template literals formatted. The large number of users was also a great way to surface obscure bugs that we would eventually hit and many different people chimed in to fix them.
+
+The best thing is that I stopped working on the project full time 6 months ago and the project kept going under the leadership of [@azz]. I'd like to thank all the people that helped in various ways, this is so exciting to make history together!
+
+Open source has been doing wonders for the project, delivering a ton of value for Facebook (almost all our JavaScript files are now pretty printed) and for the industry at large as seen by the huge adoption.
+
+### Tooling: Jest Snapshot Tests & Playground
+
+All those contributions were possible because it was easy to report errors, contribute and review code. The two tools that were most impactful in this project were jest snapshot tests and the playground.
+
+Snapshot tests are a wonder for a pretty printer. Adding new tests is dead easy, just create a file in the test folder with the code you want to format and run jest, voila! Anytime you change something, you can see how a bunch of examples would be printed differently, then it's up to you to decide if it's better or not. For a reviewer, it's also really nice, you can look at the before/after of all the things it changed. I've come to pay more attention to the snapshots than the actual implementation.
+
+[The playground](https://prettier.io/playground) is a really nice way to get to a repro or play with Prettier without having to install the development environment, being on the right branch... This has proven to be extremely valuable to get people to provide really good bug reports that are actionable.
+
+## Highlights
+
+### Introduce proper support for Vue Single File Components ([#3563]) by [@vjeux]
+
+There's a lot of demand for Vue SFC ([#2097]). We've introduced partial support for them: All the HTML is printed as is, but now the `<script>` and `<style>` tags are printed using Prettier.
+
+To use it, just run `prettier` on your `*.vue` files!
+
+### API
+
+#### Create the Prettier Plugin API ([#3536]) by [@azz]
+
+### Flow
+
+#### Print flow type annotation comments as comments ([#3449]) by [@duailibe]
+
+```ts
+// Before
+let foo: string = "a";
+
+// After
+let foo /*: string */ = "a";
```

Comment:
this example makes it seem like prettier converts normal type annotations into the comment type. (type `ts` in the code fence is sorta weird as well, but doesn't matter I suppose)

Reply:
I suppose it's the same of the other examples. Maybe add a "before prettier" variant in addition to the comparison between prettier 1.9 and 1.10?

Reply:
I don’t think there’s flow highlighting, and regular JS highlighting doesn’t highlight Flow stuff properly.

Reply:
The description bit still hasn't been written explaining that comment types will be preserved.